.mailmap

```diff
@@ -0,0 +1,2 @@
+Jörn Zaefferer <joern.zaefferer@gmail.com>
+Richard D. Worth <rdworth@gmail.com>
```

Comment:
It doesn't really matter now but why don't i get that red sign on my machine, any ideas?
![screen shot 2015-11-26 at 6 59 24 pm](https://cloud.githubusercontent.com/assets/8016851/11424076/ff0f3b98-946f-11e5-8b99-92ceebd7672a.png)